docs/index.md

```diff
@@ -0,0 +1,20 @@
+# Coder Documentation
+
+## Table of Contents
```

Comment:
We should probably add "walkthrough" to the table of contents

Reply:
Oh yeah, good call.